client/jira.go

```diff
@@ -72,7 +72,7 @@ type (
 		Issues       []domain.Issue `json:"issues"'`
```

Comment:
I'm surprised this typo hasn't caused issues. there is a single quite after double quote `json:"issues"'` can you fix this please